core/lib/zksync_core/src/eth_sender/zksync_functions.rs

```diff
@@ -19,7 +19,7 @@ pub(super) struct ZkSyncFunctions {
     pub(super) get_protocol_version: Function,
 
     pub(super) verifier_contract: Contract,
-    pub(super) get_verification_key: Function,
+    pub(super) get_verification_key: Option<Function>,
```

Comment:
This function shouldn't be optional, because new server should be able to send proofs for old batches. I think we should have its abi as a constant like we do for `commitBlocks`